src/pages/SignLoading.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand/dist'
+import { Spinner } from 'new-components/utils'
+import React from 'react'
+
+const SpinnerWrapper = styled('div')`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  font-size: 3rem;
+  color: ${colorsV3.black};
```

Comment:
Maybe move this one to utils as well? And name it differently from the previous one? To be used elsewhere maybe